modules/tiles/src/tileset/tile-3d.js

```diff
@@ -170,16 +170,32 @@ export default class TileHeader {
     }
   }
 
+  // If skipLevelOfDetail is off try to load child tiles as soon as possible so that their parent can refine sooner.
```

Comment:
JSDoc style `/** */` shows the comment when hovering in vscode.

Reply:
Fixed!